srtp.go

```diff
@@ -54,45 +54,38 @@ func (c *Context) DecryptRTP(dst, encrypted []byte, header *rtp.Header) ([]byte,
 	return c.decryptRTP(dst, encrypted, header)
 }
 
-// EncryptRTP encrypts a plaintext RTP packet, writing to the dst buffer provided.
-// If the dst buffer does not have the capacity to hold `len(plaintext) + 10` bytes, a new one will be allocated.
-// If a rtp.Header is provided, it will be Unmarshaled using the plaintext.
-func (c *Context) EncryptRTP(dst []byte, plaintext []byte, header *rtp.Header) ([]byte, error) {
-	// TODO(@lcurley) Potentially accept a *rtp.Packet to avoid this Unmarshal
```

Comment:
```
         .          .     62:	if header == nil {
         .       10ms     63:		header = &rtp.Header{}
         .          .     64:	}
         .          .     65:
         .       40ms     66:	err := header.Unmarshal(plaintext)
         .          .     67:	if err != nil {
         .          .     68:		return nil, err
         .          .     69:	}
```

This weird header unmarshalling is less of a performance bottleneck but it's something.